### DISCUSSION TITLE
Support on github for project authors to be able to support users, not discord face book or twitter?

### DISCUSSION THREAD

**Discussion by dkuchay** [community/community#20169] [Discussions]
Hello all, 

I see a large number of projects on github that I like supporting. For me, the breakage happens when I need support. Is there no way for project authors to support users here? The only way for me to get support or even ask a question is not available for me on all the projects I like here. Is the closest thing to getting support to report an issue? It's not the same. I have questions on the VzBot 3d printer that are not answered in the wiki and the only other options are discord, facebook or twitter. I work in IT and generally think that these are society's largest cesspools. If parents have to protect their kids from these venues how good do think users experiences are there? If someone offered me money I still would not go to these sites. Github is owned by microsoft. Whats the chance you guys would pipe some kind of support option into github so that those who are against using social media can actually ask a question?  My user experience goes completely south due to the lack of such an option once I realize I cant partake in a project here because no support exists that I can use.

**Comment by D7EAD** [comment]
Project maintainers have the ability to open a discussion board, much like this one, on their repositories to offer community-led support for that specific repository. If they choose not to do so, or are unaware that they can do so, then, sadly, yes, the only way to get in contact with contributors and collaborators of a given project are alternative means of communication.

Some have recommended that Github implement Direct Message functionality directly on-site and, frankly, I think this is a fantastic idea to get in direct contact with users in lieu of a Discussion board or other means of communication.

**Comment by dkuchay** [comment]
Thank you for your reply. I completely agree with the direct message feature being a fantastic idea. Didn’t know discussion boards were an option.
